### PR TITLE
Localization and specify template info added

### PR DESCRIPTION
**Pull request by ajithkumar-maragathavel**
## Purpose of this pull request

This pull request (PR) updates information on localization and also added another way of specifying templates for the blocks.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-override.html
-  https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/templates/template-override.html

### CODE REVIEW

File: src/guides/v2.3/frontend-dev-guide/templates/template-override.md [hunk 1]
```diff
@@ -93,6 +107,14 @@ is the same as writing
 <?php echo $block->getAdjustmentsHtml() ?>
 ```
 
+## Localization
+
+In order to support the translation of content, the text must be wrapped inside `__('sample text')`.
+
+```phtml
+<span><?= $block->escapeHtml(__('Back to Product Reviews')) ?></span>
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/templates/template-override.md:115]: Block version is deprecated.
```suggestion
<span><?= $escaper->escapeHtml(__('Back to Product Reviews')) ?></span>
```

File: src/guides/v2.3/frontend-dev-guide/templates/template-override.md [hunk 2]
```diff
@@ -26,6 +26,20 @@ The `category.image` block is rendered by the `image.phtml` template in the `cat
...
 
+Templates can also be specified in the PHP Block class using the `$_template` protected variable.
+
+Here is the example from `app/code/Magento/Review/view/frontend/layout/review_product_view.xml` layout:
+
+```xml
+<block class="Magento\Review\Block\View" name="review_view" ifconfig="catalog/review/active"/>
+```
+
+Template for the `review_view` block is not specified using the `template` attribute of the `<block>` layout instruction. Instead It is specified in the `app/code/Magento/Review/Block/View.php` block class.
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/templates/template-override.md:37]:
```suggestion
Template for the `review_view` block is not specified using the `template` attribute of the `<block>` layout instruction. Instead it is specified in the `app/code/Magento/Review/Block/View.php` block class.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by ajithkumar-maragathavel** [comment]
@BarnyShergold Thanks for the review. Updated the PR with the required changes.

**Comment by BarnyShergold** [comment]
@magento run all tests

**Comment by BarnyShergold** [comment]
You need to correct linting errors

**Comment by ajithkumar-maragathavel** [comment]
> You need to correct linting errors

@BarnyShergold Can you please let me how to see the error logs for lint? Because Right now I can only able to see the below information on the test result.

lint-error

**Review by BarnyShergold — APPROVED** [review]
Oddly when I came back, all the Linting Tests were working - so I'll approve

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8768`.

**Pull request merged**